phantomJS.py

```diff
@@ -0,0 +1,102 @@
+import sys, re, time
+from bs4 import BeautifulSoup
+from urlparse import urlparse
+from selenium import webdriver
+
+visitedDomains = set()
+foundEmails = set()
+clickedLinks = set()
+originDomain = ""
+
+def find_email_address2(domain, visitedPages):
+    visitedPages -= 1
+    domain = 'http://'+domain
+    if domain in visitedDomains:
+        return
+    o = urlparse(domain)
+    #verify we stay in domains that match the input netlocation
+    if originDomain not in o.netloc:
+        return
+    print("visiting: "+domain)
+    driver = webdriver.PhantomJS()
+    driver.set_window_size(1124, 850)
```

Comment:
PhantomJS is a headless driver so this simulates having an open window the size of a normal browser, without it elements may not load and other bugs such as not being able to click or interact with the page occur